Tuturial-Structure-Template.md

```diff
@@ -1,68 +1,178 @@
+> 
+> Write an intro sentence to explain the subsections:
+> 
+> For example:
+
+Step-up authentication implementation involves changes to components in your Agora Client and security infrastructure. This section shows you how to:
+
+- [Build \<module1 in procedure>](link to module1)
+- [Build \<module2 in procedure>](link to module2)
+
+> If there are no locical subsections, write a single procedure here.
+
+### Build \<module1 in procedure> 
+
+> A module explains how to implement a set of functions that operate together to produce a specific feature of the project. 
+> Be sure to explain all inputs and outputs, discuss their significance within the code and how they work together.
+> 
+> All code snippets should use: 
+> - Blog or Medium - GitHub Gists. if the code is only a snippet make sure to denote that in the file name following the format: `functionName+snippet.fileExtension`. For example if my function name is generateToken and its a javascript snippet, the Gist would have the title: `generateToken+snippet.js`. If the code within the Gist is an entire file or set of functions that can be copied and pasted and will work on their own as a library or project, name the Gist after the file they would represent with the repository.
+> - Dev.to or any markdown file - within code blocks:
+> 
+        > ```javascript 
+        > code blocks 
+        > Always remember to put the coding language 
+        > ```
+    >
+> For example:
+
+To add step-up-authentication to your Agora client:
+
+1. Do this.
+1. Do that:
+   1. Sometimes you need substeps.
+   1. Always at least 2 substeps.  
+1. Do something else.
+
+### Build \<module2 in procedure>
+
+> Brief explanation about what the technology does in this section, then a procedure.
+> 
+> For example:
+
+To add step-up-authentication to your security infrastructure:
+
+1. Do this.
+1. Do that:
+    1. Sometimes you need substeps.
+    1. Always at least 2 substeps.
+1. Do something else.
+
+
+## Test \<A rewording of the page title>
+
+To ensure that you have implemented \<A rewording of the page title> correctly:
+
+1. Do this.
+1. Do that:
+    1. Sometimes you need substeps.
+    1. Always at least 2 substeps.
+1. Do something else.
+
+## Conclusion
 
-## Conclusion ##
 Write a short paragraph in conclusion. At the end of the paragraph, please include a link to the completed source code on GitHub.  
 
-## Other Resources ## 
-> (use as section title)
+> Always include a section at the end of each post that includes links to the relevant Agora Documentation and Advanced Guides.
+
+Your next steps after implementing this functionality are:
+
+1. Read the following documentation:
+    1. Very good document about something useful.
+   1. Another very good document about something useful.
+1. Join the Agora.io Developer Slack community.
```

Comment:
Is this sentence compulsory? What's the Chinese counterpart of Slack?

Reply:
For EN only. Nice point. Thanks